### PR TITLE
Cohort Import restructuring

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #NONE

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/mixpanel.md [hunk 1]
```diff
@@ -1,5 +1,53 @@
...
+
+### Step 2: Set up the Braze integration in Mixpanel
+
+In Mixpanel, navigate to **Data Management > Integrations.** Next, select the Braze integration tab and click **Connect**. In the prompt that appears, provide the Braze data import key and REST endpoint, and click **Continue**.
+
+![]({% image_buster /assets/img_archive/mixpanel2.png %}){: style="max-width:50%;"}
+
+### Step 3: Export a Mixpanel cohort to Braze
+
+In Mixpanel, navigate to **Data Management > Cohorts.** Select the cohort to send to Braze and then select **Export to Braze**. Lastly, select a one-time sync or dynamic sync. Selecting dynamic sync will sync your Braze cohort every 15 minutes to match users in Mixpanel.
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/mixpanel.md:43]:
```suggestion
In Mixpanel, navigate to **Data Management > Cohorts**. Select the cohort to send to Braze and then select **Export to Braze**. Lastly, select a one-time sync or dynamic sync. Selecting dynamic sync will sync your Braze cohort every 15 minutes to match users in Mixpanel. 
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/mixpanel.md [hunk 2]
```diff
@@ -1,5 +1,53 @@
...
+
+![]({% image_buster /assets/img_archive/mixpanel3.png %}){: style="max-width:50%;"}
+
+### Step 4: Segment users in Braze
+
+In Braze, to create a segment of these users, navigate to **Segments** under **Engagement**, name your segment, and select **Mixpanel_Cohorts** as the filter. Next, use the "includes" option and choose the cohort you created in Mixpanel. 
+
+![In the Braze segment builder, the user attributes filter "Mixpanel cohorts" is set to "includes" and "Braze cohort".]({% image_buster /assets/img_archive/mixpanel1.png %})
+
+Once saved, you can reference this segment during Canvas or campaign creation in the targeting users step.
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/mixpanel.md:53]:
```suggestion
After saving, you can reference this segment during Canvas or campaign creation in the targeting users step.
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/amplitude.md [hunk 3]
```diff
@@ -1,5 +1,58 @@
...
+
+Cohort syncs can be set to be one-time sync, scheduled as daily or hourly, or even real-time which updates every minute. Make sure to select an option that makes sense for your business needs while also being mindful of consuming [data points]({{site.baseurl}}/user_guide/onboarding_with_braze/data_points/).
+
+### Step 4: Segment users in Braze
+
+In Braze, to create a segment of these users, navigate to **Segments** under **Engagement**, name your segment, and select **Amplitude Cohorts** as the filter. Next, use the "includes" option and choose the cohort you created in Amplitude. 
+
+![In the Braze segment builder, the filter "amplitude_cohorts" is set to "includes_value" and "Amplitude cohort test".]({% image_buster /assets/img/amplitude2.png %})
+
+Once saved, you can reference this segment during Canvas or campaign creation in the targeting users step.
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/amplitude.md:58]:
```suggestion
After saving, you can reference this segment during Canvas or campaign creation in the targeting users step.
```

File: _docs/_partners/data_and_infrastructure_agility/cohort_import/amplitude.md [hunk 4]
```diff
@@ -1,5 +1,58 @@
...
+
+{% alert note %}
+If you are using the [older navigation]({{site.baseurl}}/navigation), you can find **Technology Partners** under **Integrations**.
+{% endalert %}
+
+Once generated, you can create a new key or invalidate an existing one. The data import key and the REST endpoint are used in the next step when setting up a postback in Amplitude's dashboard.<br><br>![]({% image_buster /assets/img/amplitude3.png %})
+
+### Step 2: Set up the Braze integration in Amplitude
+
+In Amplitude, navigate to **Sources & Destinations > [project name] > Destinations > Braze**. In the prompt that appears, provide the Braze data import key and REST endpoint, and click **Save**.
```
Comment (lydia-xie) [inline _docs/_partners/data_and_infrastructure_agility/cohort_import/amplitude.md:35]:
```suggestion
In Amplitude, navigate to **Sources & Destinations** > **[project name]** > **Destinations** > **Braze**. In the prompt that appears, provide the Braze data import key and REST endpoint, and click **Save**.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
lgtm! just flagged a few style things for your review!

**Review by lydia-xie — APPROVED** [review]
Small note that there are some small inconsistencies when we call out "cohort import" in the article title:
- Heap
- Amplitude cohort import
- AppsFlyer Audiences
- Hightouch cohort import
- Kubit cohort import
- Mixpanel cohort import
- Tinyclues

Otherwise, lgtm 👍

**Comment by rachel-feinberg** [comment]
> Small note that there are some small inconsistencies when we call out "cohort import" in the article title:
> 
> * Heap
> * Amplitude cohort import
> * AppsFlyer Audiences
> * Hightouch cohort import
> * Kubit cohort import
> * Mixpanel cohort import
> * Tinyclues
> 
> Otherwise, lgtm 👍 Approving and letting Bre + Alex take a look at the PR :)

Ah yeah, so Heap and Tinyclues only reside in Cohort Import, so I didn't use the "cohort import" naming convention. Most of those with "cohort import" in their article titles have "main" articles in other sections. (Amplitude's main article is in Analytics, Hightouch's main article is in Workflow Automation, etc.)  

AppsFlyer has a specific name for cohort import (AppsFlyer Audiences), so I used that in the navigation header and article title.

**Comment by bre-fitzgerald** [comment]
@rachel-feinberg I think your approach with the naming makes sense to me. One other piece of feedback I have is how we interlink these with their "main" articles.

I think we might be able to simplify the cohort import-specific articles to make the relationship between the two clearer. What are your thoughts on cutting away some of the descriptive text at the top, and only stating that this article is for cohort import. Something like this? (excuse the squiggles, I did this with design mode)


I like the way you've phrased linking out to the cohort import articles though, that seems very clear. 👍 

cc: @lydia-xie

**Comment by rachel-feinberg** [comment]
> @rachel-feinberg I think your approach with the naming makes sense to me. One other piece of feedback I have is how we interlink these with their "main" articles.
> 
> I think we might be able to simplify the cohort import-specific articles to make the relationship between the two clearer. What are your thoughts on cutting away some of the descriptive text at the top, and only stating that this article is for cohort import. Something like this? (excuse the squiggles, I did this with design mode) 
> 
> I like the way you've phrased linking out to the cohort import articles though, that seems very clear. 👍
> 
> cc: @lydia-xie

Yes! This is so much cleaner. I'll implement this

**Review by bre-fitzgerald — APPROVED** [review]
Lgtm! Thanks for all your hard work on this 💪 🎉

**Review by alex-braze — APPROVED** [review]
Looks amazing!! :)

**Pull request merged**